src/components/Jobs/jobs.util.js

```diff
@@ -275,7 +276,10 @@ export const handleAbortJob = (
               setConfirmData,
               dispatch
             ),
-          message: 'Aborting job failed'
+          message:
+            error.response?.status === CONFLICT_ERROR_STATUS_CODE && error.response?.data?.detail
+              ? error.response.data.detail
+              : 'Aborting job failed'
```

Comment:
I believe we can always show `error.response.data.detail` if it exists and the default message otherwise.